.appveyor.yml

```diff
@@ -93,7 +93,7 @@ install:
 build_script:
     - mkdir build
     - cd build
-    - '"c:/Program Files (x86)/Microsoft Visual Studio 12.0/VC/vcvarsall" %vc_arch%'
+    - '"c:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall" %vc_arch%'
     - path
     - ps: |
         $zlibinc = c:\cygwin\bin\find /cygdrive/c/deps/ -path '*v120*/zlib.h'
```

Comment:
Yeh, this is an unsupported way to use nuget, the proper one is through visual studio GUI; I didn't know what is the format of filenames there, but needed to select a single file, without hardcoding too much.

Maybe there is a better way to use nuget from cmake-generated nmake...